lib/rust/parser/debug/tests/parse.rs

```diff
@@ -675,6 +675,16 @@ fn code_block_bad_indents1() {
     test(&code.join("\n"), expected);
 }
 
+#[test]
+fn fail_on_bad_indents() {
+    let code = ["group =", "    space4", "        space8", "     space5"];
```

Comment:
The code is:
```ruby
group =
    space4
        space8
     space5
```
e.g. block with indentation `4`, followed by nested block with indentation `8` and then block with indentation `5` - a typical sign of error.

Right now the test succeeds - the goal is to _make it fail_!

Reply:
With the 350c2c3 change I am getting a failure in following tests:
```
code_block_bad_indents1
code_block_bad_indents2
fail_on_bad_indents
```
however the failure is probably not a proper one...

Reply:
Expecting invalid node since 3e8e6c0